readme.md

```diff
@@ -14,11 +14,6 @@ This project is a serverless implementation to provide CAP XML services, through
 
 `npm test`
 
-### Running / Debugging Lambdas Locally
-
-Following Serverless V.3 reaching end of support and a decision not to [upgrade to Serverless V.4](https://wb.serverless.com/framework/docs-guides-upgrading-v4), running / debugging Lambdas is **not** supported temporarily.
-Alternative technlogies will be used to provide this functionality as soon as possible.
-
```

Comment:
I suggest retaining this until  a replacement is implemented.

Reply:
readded this place holder